spec.html

```diff
@@ -21032,23 +21032,36 @@ <h1>Static Semantics: BoundNames</h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-static-semantics-classelementkind">
+      <h1>Static Semantics: ClassElementKind</h1>
+      <emu-grammar>ClassElement : MethodDefinition</emu-grammar>
+      <emu-alg>
+        1. If PropName of |MethodDefinition| is *"constructor"*, return ~ConstructorMethod~.
+        1. Return ~NonConstructorMethod~.
+      </emu-alg>
+      <emu-grammar>ClassElement : `static` MethodDefinition</emu-grammar>
+      <emu-alg>
+        1. Return ~NonConstructorMethod~.
+      </emu-alg>
+      <emu-grammar>ClassElement : `;`</emu-grammar>
+      <emu-alg>
+        1. Return ~NotAMethod~.
```

Comment:
I would prefer to describe what it is instead of what it's not. `~empty~` or something would work.

Reply:
That seems like a good tweak.